internal/inventory/asset.go

```diff
@@ -329,9 +329,32 @@ type AssetResourcePolicy struct {
 	Condition  map[string]any `json:"condition,omitempty"`
 }
 
+// EntityMetadata maps metadata required to use Entity Store
+type entityMetadata struct {
+	Category string `json:"category"`
+	Type     string `json:"type"`
```

Comment:
Are there any other fields planned? What will happen if we change the taxonomy we have now (Category, Sub-Category, Type, Sub-type) to a less granular one?

Reply:
There might have other fields.

Once we change the taxonomy, we should follow the taxonomy.